libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss

```diff
@@ -81,6 +81,17 @@
       }
     }
 
+    &.sky-text-editor-invalid {
+      .sky-text-editor-label-wrapper {
+        color: var(--sky-highlight-color-danger);
+      }
+
+      .sky-text-editor-wrapper {
+        @include mixins.sky-theme-modern-border-invalid;
+        border-top: 1px solid $sky-border-color-neutral-medium;
```

Comment:
Why do we have the border top here? I don't see it in the focus or hover styles.